docs/lib/graphqlapi/fragments/native_common/authz/common.md

```diff
@@ -1,9 +1,29 @@
 For client authorization AppSync supports API Keys, Amazon IAM credentials, Amazon Cognito User Pools, and 3rd party OIDC providers. This is inferred from the `amplifyconfiguration.json` file when you call `Amplify.configure()`.
 
-## API Key
+#### API Key
 
-API Key is the easiest way to setup and prototype your application with AppSync.
+API Key is the easiest way to setup and prototype your application with AppSync. 
+
+#### Cognito User Pools
+
+Amazon Cognito User Pools is the most common service to use with AppSync when adding user Sign-Up and Sign-In to your application. If your application needs to interact with other AWS services besides AppSync, such as S3, you will need to use IAM credentials with Cognito Identity Pools. The Amplify CLI can automatically configure this for you when running `amplify add auth` and can also automatically federate User Pools with Identity Pools. This allows you to have both User Pool credentials for AppSync and AWS credentials for S3. You can learn more about Amplify Auth outlined in the [Accessing credentials section](~/lib/auth/access_credentials.md). For manual configuration, add the following snippet to your `amplifyconfiguration.json` file, under the `awsCognitoAuthPlugin`:
```

Comment:
> and can also automatically federate User Pools with Identity Pools

This is a tough clause and I'm not sure it explains itself fully. Can we simplify the language and express the concept more directly?

Reply:
took a stab at this...
```
Amplify CLI can automatically configure this for you when running `amplify add auth` and will also automatically use the authenticated user from user pools to federate with the identity pools to provide the AWS IAM credentials in the application. [See this for more information about the differences](https://aws.amazon.com/premiumsupport/knowledge-center/cognito-user-pools-identity-pools/)
```